style.css

```diff
@@ -106,8 +110,118 @@
     .btn{
         text-align: left;
         margin-top: 38px;
-        /*margin-right: 0;*/
-        /*margin-left: 20px;*/
+        margin-left: 20px;
         border: none;
     }
+}
+.recent_posts{
+    margin-top: 59px;
+    background-color: #EDF7FA;
+    margin-bottom: 38px;
+}
+.recent_heading{
+    display: flex;
+}
+.view{
+    display: none;
+}
+.post{
+    text-align: center;
+    font-family: 'Heebo', sans-serif;
+    font-weight: 400;
+    font-size: 18px;
+    color: #21243D;
+    margin-left: 135px;
+    margin-right: 135px;
+    margin-top: 10px;
```

Comment:
The image in the design has a backdrop color. Also, why is this blur?

![Screenshot 2023-03-13 at 2 50 23 PM](https://user-images.githubusercontent.com/38748298/224658928-38d63a82-e430-43a4-b1ba-c67adeeb0dae.png)